src/components/Accordion/Accordion.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file Accordion.
+ * @copyright IBM Security 2019
+ */
+
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import CarbonAccordion from 'carbon-components-react/lib/components/Accordion';
+
+import { getComponentNamespace } from '../../globals/namespace';
+
+const namespace = getComponentNamespace('accordion');
+
+const { defaultProps, propTypes } = CarbonAccordion;
+
+const Accordion = ({ className, alignment, ...rest }) => {
+  const isRightAligned = alignment === 'right';
+
+  const accordionClasses = classnames(namespace, className, {
+    [`${namespace}--right`]: isRightAligned,
```

Comment:
Similar to Carbon, we can avoid redundancy in these declarations by writing:

```js
[`${namespace}--${align}`]: align
```
https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/TooltipDefinition/TooltipDefinition.js#L39